app/assets/src/components/views/compare/samples_heatmap_vis.scss

```diff
@@ -1,6 +1,17 @@
 @import "~styles/themes/colors";
+@import "~styles/themes/typography";
+@import "~styles/themes/elements";
 
 .samplesHeatmapVis {
+  .plusMinusControl {
+    position: absolute;
```

Comment:
I am not particular fond of the position of the zoom... 
But in any case we should not use absolute position. If design decides that it should be next to the header, it should be part of the header component.